functions/go/fmt/README.md

```diff
@@ -0,0 +1,73 @@
+# fmt
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Format resources
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Format formats the field ordering in YAML configuration files. Field ordering
```

Comment:
D1 go/kpt-docs-styleguide
Please do an audit for this in other places

Reply:
Updated.